examples/parallelization/parallel_pendulum.py

```diff
@@ -0,0 +1,84 @@
+import biorbd
+from time import time
+
+from biorbd_optim import (
+    OptimalControlProgram,
+    ProblemType,
+    Bounds,
+    QAndQDotBounds,
+    InitialConditions,
+    ShowResult,
+)
```

Comment:
If it is okay, this file can be merged with original parallel and the number of thread selected directly in the original file, that would be better to avoid duplication